<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Öğrenci Not Hesaplama Otomasyonu</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Öğrenci Adı:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ders Adı:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vize Not:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Final Not:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>289</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHesapla">
       <property name="text">
        <string>Hesapla</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTemizle">
       <property name="text">
        <string>Temizle</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lblName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lblLesson"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lblVize"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lblFinal"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="text_Sonuc">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnCikis">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Çıkış</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCikis</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTemizle</sender>
   <signal>clicked()</signal>
   <receiver>text_Sonuc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
